picard/disc.py

```diff
@@ -19,6 +20,23 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 
+import os
+import sys
+import traceback
+
+from PyQt5 import QtCore
+
+from picard import log
+from picard.const.sys import IS_MACOS
+
+from picard.ui.cdlookup import CDLookupDialog
+
+
+# Ensure libdiscid.dylib gets loaded from app bundle
+if IS_MACOS and getattr(sys, 'frozen', False):
```

Comment:
You can also use `IS_FROZEN` from https://github.com/metabrainz/picard/blob/master/picard/const/sys.py#L30

Reply:
Thanks, changed it